adal/build.gradle

```diff
@@ -22,6 +22,7 @@ android {
 
     defaultConfig {
         minSdkVersion 14
+        //noinspection OldTargetApi
         targetSdkVersion 23
```

Comment:
Sorry, why should it be deleted? It is for the suppression of the OldTargetApi issue as it is suggested in the previous comment.